blockchain/indexers/manager.go

```diff
@@ -271,10 +271,7 @@ func (m *Manager) Init(chain *blockchain.BlockChain) error {
 		err := m.db.View(func(dbTx database.Tx) error {
 			idxKey := indexer.Key()
 			hash, height, err = dbFetchIndexerTip(dbTx, idxKey)
-			if err != nil {
-				return err
-			}
-			return nil
+			return err
```

Comment:
Yes, you are correct that was the original reason for not doing the simplification (and it is still true), however, since the greater community (and even yourself as you noted) prefers this simplification and changing it means we can add gosimple to the list of linters run on every PR, I think the tradeoff is worth it.